sparkle/platform/output.py

```diff
@@ -0,0 +1,287 @@
+        # 5: quality, 6: runtime
+        for res in val_default:
+            # TODO: status to enum
+            results.append([res[3], res[4], res[5], res[6]])
+        configuration = self.parse_string_to_dict(val_default[0][1])
+        results_default = ValidationResults(self.solver, configuration,
+                                            instance_set, results)
+
+        results = []
+        # Form: 0: solver, 1: config, 2: set, 3: instance, 4: status,
+        # 5: quality, 6: runtime
+        for res in val_conf:
+            # TODO: status to enum
+            results.append([res[3], res[4], res[5], res[6]])
+        configuration = self.parse_string_to_dict(val_conf[0][1])
+        results_conf = ValidationResults(self.solver, configuration,
+                                         instance_set, results)
+
+        cutoff_time = self.configurator.scenario.cutoff_time
+        penalty = penalty_multiplier * \
+            self.configurator.scenario.cutoff_time
+        perf_par_conf = sgrfch.get_par_performance(val_default,
+                                                   cutoff_time,
+                                                   penalty,
+                                                   objective)
+        perf_par_def = sgrfch.get_par_performance(val_conf,
+                                                  cutoff_time,
+                                                  penalty,
+                                                  objective)
+        performance = ConfigurationPerformance(perf_par_conf, perf_par_def)
+
+        results = ConfigurationResults(performance, results_default, results_conf)
+        best_config = self.parse_string_to_dict(best_config)
+        return results, best_config
+
+    def write_output(self: ConfigurationOutput) -> None:
+        """Write data into a json file."""
+        def serialize_configuration_results(cr: ConfigurationResults) -> dict:
+            return {
+                "performance": {
+                    "configured_metrics": cr.performance.configured_metrics,
+                    "default_metrics": cr.performance.default_metrics,
+                },
+                "configured_results": {
+                    "solver": cr.configured_results.solver.name,
+                    "configuration": cr.configured_results.configuration,
+                    "instance_set": cr.configured_results.instance_set.name,
+                    "result_header": cr.configured_results.result_header,
+                    "result_vals": cr.configured_results.result_vals,
+                },
+                "default_results": {
+                    "solver": cr.default_results.solver.name,
+                    "configuration": cr.default_results.configuration,
+                    "instance_set": cr.default_results.instance_set.name,
+                    "result_header": cr.default_results.result_header,
+                    "result_vals": cr.default_results.result_vals,
```

Comment:
don't use abbreviations here, this makes it difficult to understand for the user when reading the JSON file